src/Command/Core/InitCommand.php

```diff
@@ -58,19 +48,52 @@ protected function interact(InputInterface $input, OutputInterface $output)
         $styleHelper = $this->getHelper('gush_style');
         $adapters = $this->getAdapters();
 
-        if (null === $input->getOption('adapter')) {
-            $input->setOption(
-                'adapter',
-                $styleHelper->numberedChoice('Choose repository-manager', $adapters[0])
-            );
-        }
+        $input->setOption(
+            'repo-adapter',
+            $styleHelper->numberedChoice(
+                'Choose repository-manager',
+                $adapters[0],
+                GitHelper::undefinedToDefault($input->getOption('repo-adapter'))
+            )
+        );
 
-        if (null === $input->getOption('issue-tracker')) {
-            $input->setOption(
-                'issue-tracker',
-                $styleHelper->numberedChoice('Choose issue-tracker', $adapters[1])
-            );
-        }
+        $input->setOption(
+            'issue-adapter',
+            $styleHelper->numberedChoice(
+                'Choose issue-tracker',
+                $adapters[1],
+                GitHelper::undefinedToDefault($input->getOption('issue-adapter'))
+            )
+        );
+
+        $input->setOption(
+            'org',
+            $styleHelper->ask(
+                'Specify the repository organization name',
+                GitHelper::undefinedToDefault($input->getOption('org'))
+            )
+        );
+
+        $input->setOption(
+            'repo',
+            $styleHelper->ask('Specify the repository name', GitHelper::undefinedToDefault($input->getOption('repo')))
+        );
+
+        $input->setOption(
+            'issue-org',
+            $styleHelper->ask(
+                'Specify the issue-tracker organization name',
+                GitHelper::undefinedToDefault($input->getOption('issue-org'), $input->getOption('org'))
```

Comment:
The undefined is a value like `UNDEFINED_ORG = "org-autodetected\0"`, the undefinedToDefault method detects if the input value contains a null-char and if so uses the default (second parameter) instead.

Even if we use `org-autodetected` as a value someone may actually use this a repository name (its a valid name, and we can't predict usage). And the null-char makes it easier to detect the undefined ;)

When everything works as expected the user should never see this value this value publicly.